lib/bitcoin/IProofOfFeeConfig.ts

```diff
@@ -0,0 +1,22 @@
+
+/**
+ * Defines configuration of approximate sliding windows used in
+ * proof of fee calculation.
+ */
+export interface ISlidingWindowQuantileConfig {
```

Comment:
1. For sidetree, only the classic OOP interfaces are prefixed with I. For such data holder objects, you can remove the I prefix.
2. Please create a models folder and put all such models in there.
3. Please create each class in a separate file in the models folder.

Reply:
Implemented these suggestions.

Reply:
Implemented these suggestions.